lib/chef/knife/bootstrap.rb

```diff
@@ -409,7 +409,14 @@ def validate_name_args!
         end
       end
 
-      # make sure policyfile values are set correctly
+      # Ensure options are valid by checking policyfile values.
+      #
+      # The method call will cause the program to exit(1) if:
+      #   * Only one of --policy-name and --policy-group is specified
+      #   * Policyfile options are set and --run-list is set as well
+      #
+      # === Returns
```

Comment:
Can you change this  over to YARD format. I realize we're not YARD everywhere, but we're trying to slowly document the world. https://gist.github.com/chetan/1827484

Reply:
Yeah, sure. Is there a specific for exit(1) in yard?

Reply:
I think this is just @return [TrueClass] If options are valid.